esp32_code/src/Controlador.cpp

```diff
@@ -1,52 +1,56 @@
 #include "Controlador.h"
 #include "Arduino.h"
 
-Controlador::Controlador(){
-
-}
+// Construtor padrão
+Controlador::Controlador(){}
 
+// Função para ajustar o coeficiente proporcional
 void Controlador::setKp(float kp){
     Kp = kp;
 }
 
+// Função para ajustar o coeficiente diferencial
 void Controlador::setKd(float kd){
     Kd = kd;
 }
 
+// Função para ajustar o coeficiente integral
 void Controlador::setKi(float ki){
     Ki = ki;
 }
 
+// Função para ajustar todos os coeficientes do controlador
 void Controlador::setControlador(float kp, float kd, float ki){
-    setKp(kp);
-    setKd(kd);
-    setKi(ki);
+    setKp(kp); // Ajusta o coeficiente proporcional
+    setKd(kd); // Ajusta o coeficiente diferencial
+    setKi(ki); // Ajusta o coeficiente integral
```

Comment:
Chatice minha, mas é comum dar 2 espaços depois de terminar a linha do código antes de iniciar o comentário.
```suggestion
    setKp(kp);  // Ajusta o coeficiente proporcional
    setKd(kd);  // Ajusta o coeficiente diferencial
    setKi(ki);  // Ajusta o coeficiente integral
```